### PR TITLE
[accesslog] Fix for scanning long lines

### PR DESCRIPTION
**Pull request by itchyny**
This pull request fixes the accesslog plugin for scanning long lines.

### CODE REVIEW

File: mackerel-plugin-accesslog/lib/accesslog.go [hunk 1]
```diff
@@ -118,13 +118,19 @@ func (p *AccesslogPlugin) FetchMetrics() (map[string]float64, error) {
...
-	s := bufio.NewScanner(rc)
-	for s.Scan() {
+	r := bufio.NewReader(rc)
+	for {
 		var (
 			l   *axslogparser.Log
 			err error
 		)
-		line := s.Text()
+		line, err := r.ReadString('\n')
```
Comment (astj) [inline mackerel-plugin-accesslog/lib/accesslog.go]: I'm not sure what happens on logs with CR/CRLF. What do you think?

**Comment by itchyny** [comment]
Okay. ReadString does not solve the problem and possibly return `bufio.ErrBufferFull` (the test passes though). I used ReadLine.

**Comment by Songmu** [comment]
LGTM

**Comment by itchyny** [comment]
Thank you.

**Pull request merged**